test/OpenTelemetry.Instrumentation.AWSLambda.Tests/AWSLambdaWrapperTests.cs

```diff
@@ -98,32 +98,6 @@ public void TraceSyncWithInputAndNoReturn(bool setCustomParent)
             this.AssertSpanAttributes(activity);
         }
 
-        [Theory]
-        [InlineData(false)]
-        [InlineData(true)]
-        public void TraceSyncWithNoInputAndNoReturn(bool setCustomParent)
-        {
-            var processor = new Mock<BaseProcessor<Activity>>();
-
-            using (var tracerProvider = Sdk.CreateTracerProviderBuilder()
-                .AddAWSLambdaConfigurations()
-                .AddProcessor(processor.Object)
-                .Build())
-            {
-                var parentContext = setCustomParent ? CreateParentContext() : default;
-                AWSLambdaWrapper.Trace(tracerProvider, this.sampleHandlers.SampleHandlerSyncNoInputAndNoReturn, this.sampleLambdaContext, parentContext);
```

Comment:
Please, remove corresponding handler sample SampleHandlerSyncNoInputAndNoReturn.

Reply:
Done in 0e59d71